pytorch_lightning/trainer/connectors/accelerator_connector.py

```diff
@@ -102,11 +101,12 @@ def __init__(
         self._distrib_type = None
 
         self.num_processes = num_processes
-        self.tpu_cores = device_parser.parse_tpu_cores(tpu_cores)
+        # `gpus` is the input passed to the Trainer, whereas `gpu_ids` is a list of parsed gpu ids.
+        self.gpus = gpus
```

Comment:
after parsing, do we still need thsi?

Reply:
Yes! because we have Trainer properties based on gpus and gpu_ids. So, they both need to exist in Accelerator connector.